pkgs/tools/audio/headsetcontrol/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, hidapi
+}:
+
+stdenv.mkDerivation rec {
+  pname = "headsetcontrol";
+  version = "2.6";
+
+  src = fetchFromGitHub {
+    owner = "Sapd";
+    repo = "HeadsetControl";
+    rev = version;
+    sha256 = "0a7zimzi71416pmn6z0l1dn1c2x8p702hkd0k6da9rsznff85a88";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    hidapi
+  ];
+
+  configurePhase = ''
+    cmake -B "build" .
```

Comment:
Do we really need to do this manually? When adding `cmake` to `nativeBuildInputs` it should all happen automatically. Same goes for `buildPhase` and `installPhase`.